tools/peptidegenomiccoordinate/peptideGenomicCoodinate.py

```diff
@@ -0,0 +1,150 @@
+
+# 
+# Author: Praveen Kumar
+# University of Minnesota
+#
+# Get peptide's genomic coordinate from the protein's genomic mapping sqlite file (which is derived from the https://toolshed.g2.bx.psu.edu/view/galaxyp/translate_bed/038ecf54cbec)
+# 
+# python peptideGenomicCoordinate.py <peptide_list> <mz_to_sqlite DB> <genomic mapping file DB> <output.bed>
+# 
+#
+
+def main():
+    import sys
+    import sqlite3
+    conn = sqlite3.connect(sys.argv[2])
+    c = conn.cursor()
+    c.execute("DROP table if exists novel")
+    conn.commit()
+    c.execute("CREATE TABLE novel(peptide text)")
+    pepfile = open(sys.argv[1],"r")
+
+    for seq in pepfile.readlines():
+        seq = seq.strip()
+        c.execute('INSERT INTO novel VALUES ("'+seq+'")')
```

Comment:
Not a huge amount of sequences. On average it would be about 100 after all the previous proteogenomics filtering steps (I can still say that for some data it may range from under hundred to about 300 - 500 max)